src/Models/Subscription.php

```diff
@@ -201,18 +201,12 @@ public static function getActiveForThisApp() : Subscription
      * @param Users $user
      * @return Subscription
      */
-    public static function getByDefaultCompany(Users $user): Subscription
+    public static function getByDefaultCompany(Users $user)
     {
-        $subscription = self::findFirst([
+        return $subscription = self::findFirst([
```

Comment:
@rwhite27 its better to catch the exception in the controller / test. We want it to throw an internal server error if the user doesn't have a subscription